addons/l10n_ec_sale/tests/test_sale_to_invoice_ec.py

```diff
@@ -0,0 +1,78 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields
+from odoo.fields import Command
+from odoo.tests import Form, tagged
+from odoo.tools import float_is_zero
+
+from odoo.addons.sale.tests.common import TestSaleCommon
+
+
+@tagged('-at_install', 'post_install')
+class TestSaleToInvoiceEC(TestSaleCommon):
```

Comment:
```suggestion
class TestSaleToInvoiceEC(TestSaleCommon):
```
This is too specific, let's create something more generic for a small module like that.
It allows to be reused more easily if people extend the module and will therefore age better :slightly_smiling_face: 
`TestL10nECAccountSale` for example.